tests/acceptance/features/bootstrap/ChecksumContext.php

```diff
@@ -78,39 +95,55 @@ public function userHasUploadedFileToWithChecksumUsingTheAPI(
 		string $checksum
 	):void {
 		$user = $this->featureContext->getActualUsername($user);
-		$this->userUploadsFileToWithChecksumUsingTheAPI(
+		$response = $this->uploadFileToWithChecksumUsingTheAPI(
 			$user,
 			$source,
 			$destination,
 			$checksum
 		);
-		$this->featureContext->theHTTPStatusCodeShouldBeSuccess();
+		$this->featureContext->theHTTPStatusCodeShouldBeBetween(200, 299, $response);
```

Comment:
why not just 200 or 201? this is upload so should be 201?
```suggestion
		$this->featureContext->theHTTPStatusCodeShouldBeBetween(200, 299, $response);
```

Reply:
```

        /**
	 * @Then the HTTP status code should be success
	 *
	 * @return void
	 */
	public function theHTTPStatusCodeShouldBeSuccess(): void {
		$this->theHTTPStatusCodeShouldBeBetween(200, 299);
	}

```
because this is the implementaion of theHTTPStatusCodeShouldBeSuccess()

Reply:
yes that's because it was a general function that could be used for any API request but now as we know what request we are sending and what's the expected status code better to be specific

Reply:
okay